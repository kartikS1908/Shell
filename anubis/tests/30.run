./anubis tests/30.in
