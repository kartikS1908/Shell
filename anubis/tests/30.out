this is a really long pipe
this is a really long pipe
this is a really long pipe
this is a really long pipe
this is a really long pipe
this is a really long pipe
this is a really long pipe
this is a really long pipe
